packages/aura-language-server/package-lock.json

```diff
@@ -0,0 +1,10263 @@
+{
```

Comment:
We build this project using `yarn` so let's remove this lock file which is specific to `npm`